docs/components/sections/Navbar.tsx

```
@@ -0,0 +1,389 @@
+                            className={"flex items-center gap-2 no-underline text-xs"}
+                            href={"https://github.com/y-scope/clp"}
+                            rel={"noopener noreferrer"}
+                            target={"_blank"}
+                        >
+                            <svg
+                                fill={"currentColor"}
+                                height={"20"}
+                                viewBox={"0 0 24 24"}
+                                width={"20"}
+                                xmlns={"http://www.w3.org/2000/svg"}
+                            >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)